### PR TITLE
bigquery c2c docs added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+1. Enter a **Name** for the Source. The description is optional. <br/><img src={useBaseUrl('img/send-data/google-bigquery_config_main.png')} alt="google-bigquery-config-main.png" width="400" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Project ID**. Enter the unique identifier number for your project. You can find this from the Google Cloud Console.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:61]: your BigQuery project

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+1. **Delegated User Email**. Enter the admin user email address for the domain which grants the access to service account.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:62]: This field has been removed so you can take this out

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+1. **Checkpoint Field**. Enter the name of the field in the query result to be used for checkpointing. This field has to be increasing and of type number or timestamp.
+1. **Checkpoint Start**. Enter the first value for the checkpoint that the integration will plug into the query.
+1. **Time Field (Optional)**. Enter the name of the field in the query result to be parsed as timestamp. If not provided the current time will be used.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:65]: Above we use "(Option)"  before the field name so make it similar here

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+1. **Query**. Enter the BigQuery Name of the user to run. It has to include the phrase "%CHECKPOINT%".
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:66]: remove "Name of the user"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+1. **Query Interval (Optional)**. Enter the time interval to run the query in the format: Xm (for X minutes) or Xh (for X hours).
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:67]: same comment with "Optional"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+1. From the project dropdown button, select the project where you will run the BigQuery jobs.<br/><img src={useBaseUrl('img/send-data/Google_Project_Name.png')} alt="Google_Project_Name" width="400" />
+1. Click on **Create a Service Account** and follow the instructions in [Create service accounts](https://cloud.google.com/iam/docs/service-accounts-create) google cloud docs.<br/><img src={useBaseUrl('img/send-data/Google_Create_Service_Account.png')} alt="Google_Create_Service_Account" width="500" />
+1. Click on the email address provisioned during the creation and then click the **KEYS** tab.<br/><img src={useBaseUrl('img/send-data/Google_Service_Account_Keys.png')} alt="Google_Service_Account_Keys" width="600" />
+1. Click **ADD KEY**. 
+1. Click **Create new key** and choose **JSON**.
+1. Click **Create**. A JSON key file is downloaded to your computer.
+
+## States
+
+Google BigQuery integration Source is a dfully managed enterprise data warehouse that helps you to manage and analyze your data.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:33]: typo with "dfully"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+Google Cloud’s BigQuery is a fully managed enterprise data warehouse that helps you to manage and analyze your data, which also provides built-in features such as ML, geospatial analysis, and business intelligence. The Google BigQuery integration gets data from a [Google BigQuery](https://cloud.google.com/bigquery?utm_source=google&utm_medium=cpc&utm_campaign=na-none-all-en-dr-sitelink-all-all-trial-b-gcp-1605212&utm_content=text-ad-none-any-DEV_c-CRE_648217359723-ADGP_Desk%20%7C%20BKWS%20-%20BRO%20%7C%20Txt%20_%20BI%20Engine%20_%20Product%20Support-KWID_43700075197895289-kwd-1409510316386-userloc_9031936&utm_term=KW_bigquery%20bi%20engine-ST_BigQuery%20BI%20Engine-NET_g-&gclid=CjwKCAjwxr2iBhBJEiwAdXECw6GBvSq5vsHUwq6kL8X6cr4PxNIUY38sUiHseLcYIBRHB7ap8gUMrxoCYV4QAvD_BwE&gclsrc=aw.ds#section-1) table via a provided query.
+
+## Data Source
+
+The Google BigQuery Integration fetches data using [BigQuery API](https://developers.google.com/identity/protocols/oauth2/scopes#bigqueryv2).
+
+## Setup and Configuration
+
+Follow the below steps to get the Service Account's Credential JSON file to run BigQuery jobs:
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:20]: From line 20 to line 29: please see latest content in the main branch

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+1. Click on **Create a Service Account** and follow the instructions in [Create service accounts](https://cloud.google.com/iam/docs/service-accounts-create) google cloud docs.<br/><img src={useBaseUrl('img/send-data/Google_Create_Service_Account.png')} alt="Google_Create_Service_Account" width="500" />
+1. Click on the email address provisioned during the creation and then click the **KEYS** tab.<br/><img src={useBaseUrl('img/send-data/Google_Service_Account_Keys.png')} alt="Google_Service_Account_Keys" width="600" />
+1. Click **ADD KEY**. 
+1. Click **Create new key** and choose **JSON**.
+1. Click **Create**. A JSON key file is downloaded to your computer.
+
+## States
+
+Google BigQuery integration Source is a dfully managed enterprise data warehouse that helps you to manage and analyze your data.
+When you create an Rapid7 Source, it goes through the following stages:
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:34]: why "Rapid7"?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+1. **Project ID**. Enter the unique identifier number for your project. You can find this from the Google Cloud Console.
+1. **Delegated User Email**. Enter the admin user email address for the domain which grants the access to service account.
+1. **Checkpoint Field**. Enter the name of the field in the query result to be used for checkpointing. This field has to be increasing and of type number or timestamp.
+1. **Checkpoint Start**. Enter the first value for the checkpoint that the integration will plug into the query.
+1. **Time Field (Optional)**. Enter the name of the field in the query result to be parsed as timestamp. If not provided the current time will be used.
+1. **Query**. Enter the BigQuery Name of the user to run. It has to include the phrase "%CHECKPOINT%".
+1. **Query Interval (Optional)**. Enter the time interval to run the query in the format: Xm (for X minutes) or Xh (for X hours).
+1. **Google BigQuery Credential**. Upload the JSON Key file downloaded from Google Cloud IAM & Admin.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:68]: "Upload the Credential JSON file created in the previous section"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+1. **Processing Rules for Logs (Optional)**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Submit**.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:70]: "Save" not "Submit"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+
+### Error Types
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:72]: Missing the section on "On Query, Checkpoint and Checkpoint Start" - see README file in main branch.

**Review by duchatran — DISMISSED** [review]
Have some comments

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md (hunk 8 above)
Comment (JV0812) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:34]: updated.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md (hunk 11 above)
Comment (JV0812) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:130]: added

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 12]
```diff
@@ -0,0 +1,197 @@
+---
+id: google-bigquery-source
+title: Google BigQuery Source
+sidebar_label: Google BigQuery
+description: Learn how to collect data using the BigQuery API.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/google-bigquery-icon.png')} alt="google-bigquery-icon" width="70" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:10]: Can you confirm this matches the UI image? The one we use for BigQuery app is different https://help.sumologic.com/docs/integrations/google/bigquery

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 13]
```diff
@@ -0,0 +1,197 @@
...
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Project ID**. Enter the unique identifier number for your BigQuery project. You can find this from the Google Cloud Console.
+1. **Checkpoint Field**. Enter the name of the field in the query result to be used for checkpointing. This field has to be increasing and of type number or timestamp.
+1. **Checkpoint Start**. Enter the first value for the checkpoint that the integration will plug into the query.
+1. **(Optional)Time Field**. Enter the name of the field in the query result to be parsed as timestamp. If not provided the current time will be used.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:64]:
```suggestion
1. **(Optional) Time Field**. Enter the name of the field in the query result to be parsed as timestamp. If you don't, the current time will be used.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 14]
```diff
@@ -0,0 +1,197 @@
...
+1. **Query**. Enter the query that you need to run. It has to include the phrase **%CHECKPOINT%**.
+1. **(Optional) Query Interval**. Enter the time interval to run the query in the format: Xm (for X minutes) or Xh (for X hours).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:66]:
```suggestion
1. **(Optional) Query Interval**. Enter the time interval to run the query in the format: `Xm` (for X minutes) or `Xh` (for X hours).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 15]
```diff
@@ -0,0 +1,197 @@
...
+1. When you are finished configuring the Source, click **Save**.
+
+### Sample values for Query, Checkpoint, and Checkpoint Start fields
+
+Each query must contain a phrase **%CHECKPOINT%**, integration will extract and save the current checkpoint and use it in place of this phrase. The value of **Checkpoint Start** must be the same type as the **Checkpoint Field**.
+:::note
+Quote the phrase as **"%CHECKPOINT%"** if the Checkpoint Field is a timestamp string.
+:::
+
+Following are some examples which will help you understand on what value to use for the field Query, Checkpoint, Time Field and Checkpoint Start.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:78]:
```suggestion
Following are some examples that demonstrate what values to use for the Query, Checkpoint, Time Field, and Checkpoint Start fields.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md [hunk 16]
```diff
@@ -0,0 +1,197 @@
...
+
+| Field | Value |
+|:---|:---|
+| `Checkpoint Field` | `timestamp` |
+| `Checkpoint Start`| `1683053865563258` |
+| `Time Field` | `timestamp` |
+
+### Error Types
+
+When Sumo Logic detects an issue it is tracked by Health Events. The following table shows the three possible error types, the reason for the error, if the source attempts to retry, and the name of the event log in the Health Event Index.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:132]:
```suggestion
When Sumo Logic detects an issue, it is tracked by Health Events. The following table shows the three possible error types, the reason for the error, if the source attempts to retry, and the name of the event log in the Health Event Index.
```

**Review by kimsauce — DISMISSED** [review]
grammar, formatting fixes

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 17]
```diff
@@ -206,161 +206,167 @@ In this section, we'll introduce the following concepts:
 </div>
 <div className="box smallbox26 card">
   <div className="container">
-  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source"><img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="40"/><h4>Gmail Trace Logs Source</h4></a>
+  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source"><img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="40"/><h4>Gmail Trace Logs </h4></a>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:209]:
```suggestion
  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-tracelogs-source"><img src={useBaseUrl('img/send-data/gmail-trace-logs-icon.svg')} alt="Gmail" width="40"/><h4>Gmail Trace Logs</h4></a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md (hunk 12 above)
Comment (JV0812) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:10]: @kimsauce This is the icon from our stag environment. Previously we have Gmail BigQuery and this is Google BigQuery and both have different uses.

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md:10]: oh i see, thanks for confirming!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**